swift/hummingbird-framework/config.yaml

```
@@ -0,0 +1,8 @@
+framework:
+  website: https://github.com/hummingbird-project
```

better use the `github` shortcut
```suggestion
  github: hummingbird-project/hummingbird
```
this will link to https://github.com/hummingbird-project/hummingbird